views/submissions.ejs

```diff
@@ -2,7 +2,6 @@
 <html>
 
 <head>
-
 	<title>Submissions | CodeManiacs</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<link rel="stylesheet" type="text/css" href="/css/sheet.css">
```

Comment:
You have not included: `<link rel="stylesheet" type="text/css" href="/css/submissions.css">`